pkg/kubelet/server/server.go

```diff
@@ -357,6 +357,7 @@ func (s *Server) InstallDefaultHandlers(enableCAdvisorJSONEndpoints bool) {
 		cadvisormetrics.NetworkUsageMetrics: struct{}{},
 		cadvisormetrics.AppMetrics:          struct{}{},
 		cadvisormetrics.ProcessMetrics:      struct{}{},
+		cadvisormetrics.CPUTopologyMetrics:  struct{}{},
```

Comment:
This is adding metrics that weren't originally present, right?  Since we should cherrypick this change, we shouldn't introduce new metrics here